Assets/Scripts/UI/InGameUI/DevConsole/DevConsoleCoreCommands.cs

```diff
@@ -236,5 +236,68 @@ public static Tile GetTileAt(Vector3 pos)
         {
             return GetCurrentWorld().GetTileAt((int)pos.x, (int)pos.y, (int)pos.z);
         }
+
+        public static void SetRoomGas(int roomID, string gas, float pressure)
+        {
+            // Adding gas to room
+            Room room = World.Current.RoomManager[roomID];
+            room.SetGas(gas, pressure * room.TileCount);
+        }
+
+        public static void SetAllRoomsGas(string gas, float pressure)
+        {
+            // Adding gas to all rooms
+            foreach (Room room in World.Current.RoomManager)
+            {
+                if (room.ID > 0)
+                {
+                    room.SetGas(gas, pressure * room.TileCount);
+                }
+            }
+        }
+
+        public static void FillRoomWithAir(int roomID)
+        {
+            // Adding air to room
+            Room room = World.Current.RoomManager[roomID];
+            room.SetGas("O2", 0.2f * room.TileCount);
+            room.SetGas("N2", 0.8f * room.TileCount);
+        }
+
+        public static void FillAllRoomsWithAir()
```

Comment:
The latest commit takes care of this. The Gas commands can still overfill, but that's sort of the intent, to allow the user to bit by bit, build up the atmosphere of a given room as they please.